test/features/page_objects/poa.rb

```diff
@@ -0,0 +1,158 @@
+require_relative '../utils/index.rb'
+
+class POA
+
+  def initialize(driver)
+    @driver = driver
+  end
+
+  def file_upload
+    element = @driver.find_element(:css, '.onfido-sdk-ui-Uploader-uploadArea input[type="file"]')
+    @driver.execute_script("return arguments[0].setAttribute('style','display: true');", element)
+    @driver.execute_script("return arguments[0].value = '';", element) unless element.attribute('value').empty?
+    element
+  end
+
+  def uploaded_image
+    @driver.find_element(:css, '.onfido-sdk-ui-Confirm-CaptureViewer-image')
+  end
+
+  def verify_uk_address_title
+    @driver.find_element(:xpath, "//span[contains(.,'Let’s verify your UK address')]")
```

Comment:
What's the benefit of using xpath versus targeting the class like we do for other titles? Is this faster?

Reply:
The benefit of using xpath is that it also checks the value of the UI element which class doesn't. It adds extra assurance. 

Speed seems unchanged.

Reply:
I see. Good to know. It's the first time I see this syntax